modules/control/controller/mpc_controller.cc

```diff
@@ -240,11 +240,15 @@ Status MPCController::ComputeControlCommand(
   SimpleMPCDebug *debug = cmd->mutable_debug()->mutable_simple_mpc_debug();
   debug->Clear();
 
+  ComputeLongitudinalErrors(&trajectory_analyzer_, debug);
+
   // Update state
   UpdateStateAnalyticalMatching(debug);
 
   UpdateMatrix();
 
+  FeedforwardUpdate(debug);
```

Comment:
It is not good to pass a debug as the function parameters in those major functions. You can refer planning's design (task/dp_poly_graph/dp_road_graph.h/cc) for details.

Reply:
@lianglia-apollo Not serving the save purpose as planning do. But thanks for pointing out and will consider a redesign of the debug system later.

Reply:
@lianglia-apollo Also to keep the code consistent with the reset of code as of now.